### DISCUSSION TITLE
be able to view Friends profiles

### DISCUSSION THREAD

**Discussion by KillerParticles** [ciderapp/Cider#686] [Rejected Features]
Currently in Cider you cannot view a friends profile, clicking on their name from a playlist of theirs or other means does nothing. This sucks for many reasons but the main one is if I want to play a playlist from them, I pretty much can't unless I find it from the 'Friends listening too' section. 

Add the feature for us to view friends profiles.

**Comment by tva2000hn** [comment]
Cant, API limitation

**Comment by Amaru8** [comment]
Rejecting due to **vapormusic**'s response.
